lib/api-helper/goal/storeGoals.ts

```diff
@@ -29,26 +29,28 @@ import {
 import { Parameterized } from "../../api/goal/GoalWithFulfillment";
 import { SdmGoalEvent } from "../../api/goal/SdmGoalEvent";
 import {
-    GoalRootType,
     SdmGoalFulfillment,
     SdmGoalFulfillmentMethod,
     SdmGoalKey,
     SdmGoalMessage,
     SdmProvenance,
 } from "../../api/goal/SdmGoalMessage";
-import {
-    GoalSetRootType,
-    SdmGoalSetMessage,
-} from "../../api/goal/SdmGoalSetMessage";
+import { SdmGoalSetMessage } from "../../api/goal/SdmGoalSetMessage";
 import { GoalImplementation } from "../../api/goal/support/GoalImplementationMapper";
 import { GoalSetTag } from "../../api/goal/tagGoalSet";
 import {
     OnPushToAnyBranch,
     PushFields,
     SdmGoalState,
-    SdmGoalWithPushFields,
+    UpdateSdmGoalMutation,
+    UpdateSdmGoalMutationVariables,
+    UpdateSdmGoalSetMutation,
+    UpdateSdmGoalSetMutationVariables,
 } from "../../typings/types";
 
+// tslint:disable-next-line:no-var-requires
+const omitEmpty = require("omit-empty");
```

Comment:
Why not `import`? It seems the types are available.

Reply:
For whatever reason I couldn’t get the types to work. I can try again but I think they are broken. I’ll try again.

Reply:
Like I said, not a big deal. If you are not using them, it might be worth uninstalling them.